frontend/src/views/TemplateForm.vue

```diff
@@ -6,7 +6,9 @@
             <b-button @click="previewTemplate"
               class="is-pulled-right" type="is-primary"
               icon-left="file-find-outline">{{ $t('templates.preview') }}</b-button>
-
+            <b-button @click="previewTemplate"
```

Comment:
both buttons do `previewTemplate` on click?

Reply:
Oh, that was old code from when I thought it could be done in the editing window.